tests/checkers/unittest_stdlib.py

```diff
@@ -49,10 +49,8 @@ def test_deprecated_no_qname_on_unexpected_nodes(self) -> None:
         """
 
         def infer_func(
-            node: Name, context: Optional[Any] = None
-        ) -> Iterator[
-            Union[Iterator, Iterator[AssignAttr]]
-        ]:  # pylint: disable=unused-argument
+            node: Name, context: Optional[Any] = None  # pylint: disable=unused-argument
+        ) -> Iterator[Union[Iterator, Iterator[AssignAttr]]]:
```

Comment:
This actually make the formatting better, I like this. But it also means some work when upgrading 😿